Applications/SystemMonitor/ProcessModel.cpp

```diff
@@ -123,6 +123,10 @@ String ProcessModel::column_name(int column) const
         return "Pledge";
     case Column::Veil:
         return "Veil";
+    case Column::WorkingDirectory:
+        return "PWD";
```

Comment:
The commit message calls this `CWD`, and here you call it `WorkingDirectory` but also `PWD` (?)
Let's settle on a single name. Some options:
* Directory
* WorkingDirectory
* CWD

Reply:
It should be CWD.